package.json

```diff
@@ -28,6 +28,7 @@
     "@actions/exec": "^1.1.1",
     "@actions/glob": "^0.4.0",
     "@actions/io": "^1.1.3",
+    "github-actions.cache-buildjet": "^0.2.0",
```

Comment:
I guess this is the same as `"@actions/buildjet-cache": "npm:github-actions.cache-buildjet@0.1.2",` above, just using a different name. Either way, that change should be done as its own PR, and not intermingled here.

Reply:
Thanks, I've removed this.